server/data/weidu-tp2-base.yml

```diff
@@ -350,9 +350,10 @@ tp2-action:
         newMUSFile is copied to the music directory and added to songlist.2da under the name internalMusicName. The variable %internalMusicName% is set to the number of this new entry in songlist.2da. If newMUSFile already exists, the variable %internalMusicName% is set to the existing entry in songlist.2da and all other operations are skipped. Variables are evaluated. Some versions of the Infinity Engine cannot handle more than 100 entries in songlist.2da. ADD_MUSIC will fail under those circumstances unless the appropriate versions of ToB Hacks or ToBEx are installed or the game is GemRB or BGEE.
 
     - name: ADD_PROJECTILE
-      detail: ADD_PROJECTILE modpath/PROName.PRO
+      detail: ADD_PROJECTILE modpath/PROName.PRO idsName
       doc: |-
         Appends an entry for PROName to PROJECTL.IDS and assigns it the next available ProRef number. Then copies the file modpath/PROName.PRO to the override folder. The new ProRef number can be accessed through the variable %PROName% and used to updated the Projectile type field of an ITM or SPL file's Item Ability or Spell Ability sub-structures.
+        If `idsName` is specified, then it will give the projectile the name `idsName` in `MISSILE.IDS`.
```

Comment:
Where'd you get this from?

Reply:
> Where'd you get this from?

[Undocumented feature](https://github.com/WeiDUorg/weidu/issues/224)